grafana/dashboards/cmode/harvest_dashboard_lun.json

```diff
@@ -805,7 +805,7 @@
       "targets": [
         {
           "exemplar": false,
-          "expr": "topk($TopResources, lun_avg_write_latency{datacenter=\"$Datacenter\",cluster=\"$Cluster\",svm=~\"$SVM\",volume=~\"$Volume\",lun=~\"$LUN\"})/1000",
```

Comment:
why to remove topk from table?

Reply:
good catch, that should have been reverted